### ISSUE TITLE
Flameshot does not start at system startup Ubuntu 20.4

### ISSUE THREAD

**Issue by cytorak87** [flameshot-org/flameshot#1197]
Hi, I'm using Flameshot version 0.6.0 Compiled with Qt 5.12.5. My version Qt - `Using Qt version 5.12.8` 
I got a problem when I installed a flameshot on ubuntu 20.4. I have set the "Launch at system startup" item, but it does not work. I found one of the solutions for Ubuntu 20.4 here. I have installed `qt5-qmake qt5-default qttools5-dev-tools`. But after reboot it didn't help. Please help me solve this problem. For installation, I used the standard method `sudo apt install flameshot`

**Comment by mmahmoudian** [comment]
@cytorak87 the Flameshot version you are trying to install is very old. The most recent version at this point is 0.8.5 which you can get from [here](https://github.com/flameshot-org/flameshot/releases/latest)

Based on [repology.org](https://repology.org/project/flameshot/versions) the latest available Flameshot in [Ubuntu 20.04 is 0.8.3](https://packages.ubuntu.com/source/groovy/flameshot). Therefore I cannot comprehend how the `apt install flameshot` has given you version 0.6.0 ! Regardless, as Ubuntu user you have the following options:
1. install the 0.8.3 from `apt` (Still I'm puzzled how that has failed for you)
2. install from the `.deb` file. For this you should download the file and install it:
    ```sh
    wget "https://github.com/flameshot-org/flameshot/releases/download/v0.8.5/flameshot-0.8.5-1.ubuntu-20.04.amd64.deb"
    sudo dpkg -i "flameshot-0.8.5-1.ubuntu-20.04.amd64.deb"
    # you can later uninstall by:
    # sudo apt remove flameshot
    ```
3. install from snap:
    ```sh
    sudo snap install flameshot
    ```
4. install from Flatpak:
    ```sh
    flatpak install flathub flameshot
    ```
5. install AppImage:
    ```sh
    # Naviaget to the folder you would like to store the software (the following is a suggestion)
    mkdir -p ~/Applications/Flameshot
    cd ~/Applications/Flameshot

    # Download the latest AppImage
    sh curl -O -J -L $(curl -s https://api.github.com/repos/flameshot-org/flameshot/releases/latest \
                                | grep -Po 'https://github.com/flameshot-org/flameshot/releases/download/[^}]*\.AppImage' \
                                | uniq)
    # Set it to executable
    chmod +x Flameshot-*-x86_64.AppImage
    
    # Now you have the Flameshot ready and you can run the software
    ./Flameshot-*-x86_64.AppImage
    ```

**Comment by cytorak87** [comment]
@mmahmoudian Thanks a lot for your help! I did it thanks to you! The issue can be closed :+1:
